src/components/PopularMovies/moviePrivider.jsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable react-refresh/only-export-components */
 import { createContext, useState } from 'react';
-import options from '../../Options/Option/utils';
+import options from '../Options/Option/data';
```

Comment:
let's use absolutes imports for all imports in the project
```suggestion
import options from 'src/compnents/Options/Option/data';
```